tests/20-packet-parsing/packet-injector/project-conf.h

```diff
@@ -0,0 +1,6 @@
+#define LOG_CONF_LEVEL_IPV6                        LOG_LEVEL_DBG
+#define LOG_CONF_LEVEL_RPL                         LOG_LEVEL_DBG
+#define LOG_CONF_LEVEL_6LOWPAN                     LOG_LEVEL_DBG
+#define LOG_CONF_LEVEL_TCPIP                       LOG_LEVEL_DBG
+#define LOG_CONF_LEVEL_MAC                         LOG_LEVEL_DBG
+#define LOG_CONF_LEVEL_FRAMER                      LOG_LEVEL_DBG
```

Comment:
Seems to be a missing newline at the end of this file.

If this code is built on SimpleLink, add a workaround for `LOG_CONF_LEVEL_FRAMER` similar to commit afac9c18a8361 so the CI logs aren't filled with warnings.

Reply:
Newline and SimpleLink work-around added.